python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -275,15 +275,21 @@ def start(self,
             if initialization_hook:
                 self._initialization_hook = initialization_hook
                 self.worker_group.execute(initialization_hook)
-            if self._num_gpus_per_worker > 0:
-                self._setup_gpus()
+
+            share_cuda_visible_devices_enabled = env_bool(
```

Comment:
Nit: Can we have this be a 0 or 1 integer instead? This follows the same pattern that we currently have for SGD and Tune env vars

Reply:
Updated! The conversion logic is not as clean as I'd like but will perform as expected.